### DISCUSSION TITLE
Opentelemetry support for synchronous Gauges

### DISCUSSION THREAD

**Discussion by tauseef-ah** [open-telemetry/opentelemetry-specification#2707] [Q&A] [answered]
Why doesnt OpenTelemetry provide support for Synchronous Gauges?
Although asynchronous seems efficient to handle the sampled value, there would still exist use cases where-in the observable sample value is provided by a library, and it would have been easier to push the value to Metric collector, then to save the same value and provide it when the callback function is called.

**Comment by jmacd** [comment, marked as answer]
See #2318
